veg2hab/vegkartering.py

```diff
@@ -1196,15 +1196,20 @@ def bepaal_mozaiek_habitatkeuzes(self, max_iter: int = 20) -> None:
 
         Reviseert de habitatkeuzes op basis van mozaiekregels.
         """
-        # We starten alle HabitatKeuzes op None, en dan vullen we ze steeds verder in
-        self.gdf["HabitatKeuze"] = self.gdf.VegTypeInfo.apply(
-            lambda voorstellen_list: [None for sublist in voorstellen_list]
-        )
-
-        # TODO: hieronder de naam van de tool invoeren ipv bepaal_mits_habitatkeuzes zodat de gebruiker er ook wat aan heeft
         assert (
             "HabitatKeuze" in self.gdf.columns
-        ), "Er is geen kolom met HabitatKeuze (draai eerst bepaal_mits_habitatkeuzes)"
+        ), "Er is geen kolom met HabitatKeuze (draai eerst tool 3)"
```

Comment:
Ik denk dat het goed is als we dit nog wat explicieter maken, want nu kun je wel stap 4 overslaan toch? Of is dat acceptabel?

Reply:
Hij functioneert nu wel als je stap 4 overslaat maar ik zou niet weten waarom je dat zou doen. Het scheelt misschien wat tijd als je geen mozaiekregels bent tegengekomen, maar dan moet je dus eerst checken dat je die niet hebt. En dan nog scheelt het amper want dan is stap 4 toch heel snel, want je hebt geen mozaiekregels.

Ik ben wel voor iets explicieters, hebben we het nog over